core/src/main/scala/spinoco/fs2/cassandra/options.scala

```diff
@@ -121,23 +123,19 @@ case class QueryOptions(
     copy(tracing = Some(true))
 
   /**
-    * Sets the retry policy to use for this query.
-    * The default retry policy, if this method is not called, is the one returned by
-    * {@link com.datastax.driver.core.policies.Policies#getRetryPolicy} in the
-    * cluster configuration. This method is thus only useful in case you want
-    * to punctually override the default policy for this request.
+    * Sets the execution profile name for the request
     */
-  def withRetryPolicy(policy:RetryPolicy):QueryOptions =
-    copy(retryPolicy = Some(policy))
+  def withExecutionProfileName(name: String):QueryOptions =
```

Comment:
So no retry policy anymore on the request level ?

Reply:
Retry policies can be set at the query builder level, but not on a built query.